installinator/src/mock_peers.rs

```diff
@@ -12,10 +12,9 @@ use std::{
 use anyhow::{bail, Result};
 use async_trait::async_trait;
 use bytes::Bytes;
-use installinator_artifact_client::ClientError;
+use installinator_artifact_client::{ClientError, ResponseValue};
```

Comment:
@sunshowers this fixed a problem I hit while testing; seemed like a good enough fix, but let me know if you think differently.

Reply:
Seems fine to me